<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoShoutOut</class>
 <widget class="QWidget" name="AutoShoutOut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>287</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox { font-weight: bold; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="configGroup">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Configuration</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QSpinBox" name="cooldown_spin_box">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cooldown_combo_box">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="message_label">
            <property name="text">
             <string>Message</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="PlainTextEdit" name="message_text_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Follow {name}, last game played was {game}. https://twitch.tv/{login}</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="message_alt_label">
            <property name="text">
             <string>Message Alt</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="PlainTextEdit" name="message_alt_text_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="cooldown_check_box">
            <property name="text">
             <string>Cooldown</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Raids</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="raids_enabled_check_box">
            <property name="text">
             <string>Enable Raids</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Min Viewers</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="raid_min_viewers_spin_box">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="filteringGroup">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Filtering</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="1" column="0">
         <widget class="QCheckBox" name="whitelist_enabled_checkbox">
          <property name="text">
           <string>Whitelist</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="whitelist_line_edit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string notr="true">twitch</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="blacklist_enabled_checkbox">
          <property name="text">
           <string>Blacklist</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="blacklist_line_edit">
          <property name="placeholderText">
           <string notr="true">streamelements, streamlabs</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Enable For</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="affiliate_enabled_check_box">
            <property name="text">
             <string>Affiliate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="partner_enabled_check_box">
            <property name="text">
             <string>Partner</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>.</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
